addons/web/static/src/views/fields/field.js

```diff
@@ -104,6 +104,13 @@ export class Field extends Component {
             const value = evaluateExpr(decorations[decoName], evalContext);
             classNames[`text-${decoName}`] = value;
         }
+        const additionalClassNames =
+            fieldInfo.rawAttrs && fieldInfo.rawAttrs.class && fieldInfo.rawAttrs.class.split(" ");
+        if (additionalClassNames) {
+            for (const className of additionalClassNames) {
+                classNames[className] = true;
+            }
+        }
```

Comment:
Are you sure this is the right place to do that? This would have an impact in all views. I quickly checked in legacy, and we completely ignore the class attribute set on fields in form and kanban. In wowl, we still ignore it in form, but not in kanban. I'm not sure it's what we want in the long term, but considering the current state of master, I would encourage a conservative approach, i.e. replicating the legacy behavior. So this should probably be done in list renderer. What do you think?

Reply:
Not sure why we would avoid setting the style on the main field container if a style is explicitly set on the field from the arch (in other types of view). For me, it would be consistent to expect the same behavior in all views

Reply:
@aab-odoo In form, It seems they are not ignored. They are set on the field because of the ViewCompiler (compileNode -> copyAttributes)
Can't we generalize this behavior in the field as I did it? (and adapt if needed)